openslides/core/views.py

```diff
@@ -213,3 +214,85 @@ class CustomSlideDeleteView(CustomSlideViewMixin, utils_views.DeleteView):
     Delete a custom slide.
     """
     pass
+
+
+class TagException(Exception):
```

Comment:
Can you use a subclass of OpenSlidesError? Can you move this definition into a separate file `exceptions.py`?